code/game/objects/items/weapons/melee/melee_misc.dm

```diff
@@ -39,6 +39,10 @@
 	hitsound = 'sound/weapons/rapierhit.ogg'
 	materials = list(MAT_METAL = 1000)
 	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | ACID_PROOF // Theft targets should be hard to destroy
+	// values for slapping
+	var/slap_sound = 'sound/effects/woodhit.ogg'
+	var/slap_cooldown = (4 SECONDS)
```

Comment:
This can just be integrated into the code, I doubt anyone will want to change the slap time